src/interaction/sessionrecording.cpp

```diff
@@ -465,10 +469,10 @@ bool SessionRecording::startPlayback(std::string& filename,
     global::eventEngine->publishEvent<events::EventSessionRecordingPlayback>(
         events::EventSessionRecordingPlayback::State::Started
     );
-    initializePlayback_triggerStart();
-
     global::navigationHandler->orbitalNavigator().updateOnCameraInteraction();
 
+    initializePlayback_triggerStart();
```

Comment:
Why move this from before to after the `updateOnCameraInteraction` call?

Reply:
No real reason, apart from that it was moved and grouped together with some other function calls somewhere in the middle of the process of this PR. I'll undo this change